deploy-local.sh

```diff
@@ -273,11 +273,7 @@ CoinMarketCap:
   Token: ${CMC_TOKEN}
 
 MemCache:
-  AccountExpiration: 200
```

Comment:
If we cannot control the expiration time, user will not find the latest data if the cache is not removed. For example, `account` data will be refreshed after `AccountExpiration`, then user can get the latest account data.

Reply:
Then I think I may change to other libs because golang-lru not support expiration